packages/create/templates/hello-nextjs/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "hello-nextjs",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@dbos-inc/dbos-sdk": "^1.29.15",
+    "next": "15.0.3",
+    "oracledb": "^6.6.0",
+    "pg-query-stream": "^4.7.1",
+    "react": "19.0.0-rc-66855b96-20241106",
+    "react-dom": "19.0.0-rc-66855b96-20241106",
+    "typeorm": "^0.3.20",
+    "webpack": "^5.96.1"
+  },
```

Comment:
I think `oracledb` and `typeorm` are artifacts from working on the build order? i.e. they shouldn't be dependencies?